client/views/room/contexts/RoomContext.ts

```diff
@@ -1,15 +1,22 @@
 import { createContext, useContext } from 'react';
 
 import { IRoom, IOmnichannelRoom, isOmnichannelRoom } from '../../../../definition/IRoom';
+import { IOmnichannelSubscription } from '../../../../definition/ISubscription';
```

Comment:
I don't think we need this anymore, even because the field present in `IOmnichannelSubscription` -> `department` will no longer be returned.